### PR TITLE
Add Granite and Holocene fork references

### PR DESCRIPTION
**Pull request by sebastianst**

### CODE REVIEW

File: op-node/rollup/types.go [hunk 1]
```diff
@@ -410,12 +424,14 @@ func (c *Config) IsFjord(timestamp uint64) bool {
 	return c.FjordTime != nil && timestamp >= *c.FjordTime
 }
 
-// IsFjordActivationBlock returns whether the specified block is the first block subject to the
-// Fjord upgrade.
-func (c *Config) IsFjordActivationBlock(l2BlockTime uint64) bool {
-	return c.IsFjord(l2BlockTime) &&
-		l2BlockTime >= c.BlockTime &&
-		!c.IsFjord(l2BlockTime-c.BlockTime)
```
Comment (sebastianst) [inline op-node/rollup/types.go:418]: I reordered it to further down below, it was at a random place.

File: op-node/rollup/superchain.go [hunk 2]
```diff
@@ -86,6 +86,7 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 		DeltaTime:              chConfig.DeltaTime,
 		EcotoneTime:            chConfig.EcotoneTime,
 		FjordTime:              chConfig.FjordTime,
+		GraniteTime:            chConfig.GraniteTime,
```
Comment (protolambda) [inline op-node/rollup/superchain.go:89]: No holocene here?

File: op-node/rollup/types.go [hunk 3]
```diff
@@ -322,6 +330,12 @@ func (cfg *Config) Check() error {
 	if err := checkFork(cfg.EcotoneTime, cfg.FjordTime, Ecotone, Fjord); err != nil {
 		return err
 	}
+	if err := checkFork(cfg.FjordTime, cfg.GraniteTime, Fjord, Granite); err != nil {
+		return err
+	}
+	if err := checkFork(cfg.GraniteTime, cfg.HoloceneTime, Granite, Holocene); err != nil {
```
Comment (protolambda) [inline op-node/rollup/types.go:336]: And holocene?

**Review by Inphi — APPROVED** [review]

File: op-node/rollup/superchain.go (hunk 2 above)
Comment (sebastianst) [reply op-node/rollup/superchain.go:89]: It's still missing in the SCR. Will be added later. Right now, we just need to unblock granite.

File: op-node/rollup/types.go (hunk 3 above)
Comment (sebastianst) [reply op-node/rollup/types.go:336]: Nothing comes after holocene. Or do you want to add interop? We didn't have anything after Fjord here so I didn't put anything after Holocene.

**Review by protolambda — APPROVED** [review]

File: op-node/rollup/chain_spec.go [hunk 4]
```diff
@@ -46,6 +48,8 @@ var nextFork = map[ForkName]ForkName{
 	Delta:    Ecotone,
 	Ecotone:  Fjord,
 	Fjord:    Interop,
```
Comment (Inphi) [inline op-node/rollup/chain_spec.go:50]: whoops. This should be Fjord: Granite.

Comment (sebastianst) [reply op-node/rollup/chain_spec.go:50]: indeed 🙈 thanks for catching

**Pull request merged**